folsom_poller_test.go

```diff
@@ -105,3 +106,36 @@ func TestPollHistogram(t *testing.T) {
 		return
 	}
 }
+
+var sampleETS1 = `{"57397":{"read_concurrency":"false","write_concurrency":"true","compressed":"false","memory":2065,"owner":"'<0.1076.0>'","heir":"none","name":"folsom_slide_uniform","size":83,"node":"node","named_table":"false","type":"set","keypos":1,"protection":"public"},"53300":{"read_concurrency":"false","write_concurrency":"true","compressed":"false","memory":2481,"owner":"'<0.1076.0>'","heir":"none","name":"folsom_slide_uniform","size":122,"node":"node","named_table":"false","type":"set","keypos":1,"protection":"public"}}`
+
+var sampleETS2 = `{"syslog_tab":{"read_concurrency":true,"write_concurrency":false,"compressed":false,"memory":385,"owner":"<0.826.0>","heir":"none","name":"syslog_tab","size":6,"node":"node","named_table":true,"type":"set","keypos":1,"protection":"public"},"49196":{"read_concurrency":false,"write_concurrency":false,"compressed":false,"memory":339,"owner":"<0.813.0>","heir":"<0.815.0>","name":"gr_manager","size":3,"node":"node","named_table":false,"type":"set","keypos":1,"protection":"private"}}`
+
+func TestDecodeBody(t *testing.T) {
```

Comment:
I don't understand this test as decodeBody is json.Decoder.Decode() and that's it, so your testing a stdlib type/function

Reply:
@freeformz we have an ongoing incident that is blocked by a struct that s not being decoded correctly here. We've updated the struct to remove the offensive fields, and added a test to assure that the data we get back from Folsom is properly decoded into a `FolsomEts` struct.

If you prefer, we can certainly write a test that just decodes the struct without calling `decodeBody`, the goal is to get this out ASAP (and follow up later as needed) so please advise on what changes you want as this is now blocked on your review.